src_web/kamaho-shokusu/src/Policy/ContactsPolicy.php

```
@@ -16,19 +16,19 @@
 final class ContactsPolicy
 {
     /** お問い合わせフォーム（全認証ユーザー） */
-    public function canIndex(?IdentityInterface $user, mixed $resource): bool
+    public function canIndex(?IdentityInterface $user, \App\Controller\ContactsController $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)